### PR TITLE
Boudrey/maya 113815/centos8

### PR DESCRIPTION
**Pull request by boudrey**

### CODE REVIEW

File: cmake/googletest.cmake [hunk 1]
```diff
@@ -24,6 +24,13 @@ macro(fetch_googletest)
         # Attempting to do so in execute_process fails with string invalid escape
         # sequence parsing errors.  PPT, 22-Nov-2018.
         file(TO_CMAKE_PATH ${CMAKE_MAKE_PROGRAM} CMAKE_MAKE_PROGRAM)
+
+        # Force the use of ABI version 0 on Linux.
+        # This is what Maya has been using for 2019...2023
+        if(UNIX AND NOT APPLE)
```
Comment (seando-adsk) [inline cmake/googletest.cmake:30]: We use a more modern cmake check:
if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")

File: cmake/googletest_download.txt.in [hunk 2]
```diff
@@ -14,14 +14,6 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(FORCE_SHARED_CRT "")
-set(FORCE_OLD_ABI "")
-if(MSVC)
-  set(FORCE_SHARED_CRT "-DFORCE_SHARED_CRT=OFF")
```
Comment (seando-adsk) [inline cmake/googletest_download.txt.in:20]: I get the change for Linux (CentOS 8), but why this change to Windows?

Comment (boudrey) [reply cmake/googletest_download.txt.in:20]: It is the same issue; this line was never taken into account so I decided to remove it.

**Review by seando-adsk — APPROVED** [review]
All good thanks.
